.github/ISSUE_TEMPLATE/2-enhancement.yml

```diff
@@ -0,0 +1,33 @@
+name: ✨ Enhancement Request
+description: If you have an idea to improve an existing feature in core or need something for development (such as a new hook) please let us know or submit a Pull Request!
+title: "[Enhancement]: "
+labels: ["type: enhancement"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ### Thanks for contributing!
+
+        Please provide us with the information requested in this form. 
+
+        Make sure to look through [existing issues](https://github.com/woocommerce/woocommerce/issues?q=is%3Aopen+is%3Aissue+label%3A%22type%3A+enhancement%22) to see whether your idea is already being discussed.
+        Feel free to contribute to any existing issues.
```

Comment:
I wonder if we should also mention our [votes needed/enhancements list](https://github.com/woocommerce/woocommerce/issues?q=is%3Aissue+sort%3Areactions-%2B1-desc+is%3Aclosed+label%3A%22votes+needed%22), wdyt?

Reply:
Good idea, added a mention and link.